.gitignore

```diff
@@ -8,3 +8,5 @@ firmware-*.json
 firmware-*.sha2
 *.hex
 
+*.pyc
```

Comment:
This belongs in a global gitignore file for workspace hygiene
https://github.com/solokeys/solo-python/blob/6915722f9bd8a25f3eab982cbcc1544666bc10e5/.gitignore#L1-L3
But I suspect that is an uphill battle I will lose :grin: